### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_QueueTable_Screenreader : Proper information regarding the data in the table content is not announced to the screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/AzureStorageExplorer#3508]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#Azure Tools Storage Explorer;#WCAG1.3.1;#WCAG4.2.3;#MAC;#A11ySev1;#Benchmark;#Screenreader;#VoiceOver;#FTP;#BM-Azure Tools Storage Explorer-MAC-Aug2020;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.15.0
AzCopy Version: 10.6.0
OS Version: macOS Catalina Version 10.15.6
Screen Reader: Voiceover

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it.
6. Start screen reader and navigate to table content.
7. Observe if the proper information about the number of rows and columns and the association of data with the table header is announced or not.

### Actual:
When screen reader users navigate using 'Caps + arrow keys' to the table content it is announced as 'table with 0 column and 1 row' which is incorrect as per the visual appearance of the table. Additionally, on navigating through the data of the table the association between the data and the table header is not announced to the screen reader users.

This issue is observed in the below scenarios:
1. Blob Container Table - Table
2. FileShares - Filesharetable
3. Disk - DiskInfoTable

### Expected:
For screen reader users, proper information about the number of rows and columns of the table should be announced to them. Additionally, the association between the data and the table header should also be announced.

### User Impact:
If the screen reader users are not conveyed with proper information about the content that is present on the screen then they will miss out on the information that is available for the visual users.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-manai_microsoft_com/EdpOXANZ-H9Ejaud9pB-0ToByg1lpWtXvwuHdifiEmrJVQ?e=9Xjbvj

**Comment by craxal** [comment]
@MayuriNaik I am having trouble reproducing. My screen reader says very different things than what yours does. Can you please elaborate on:
- The settings you have configured for your screen reader.
- The key combinations you are using to navigate.

![image](https://user-images.githubusercontent.com/3473026/92822530-1d87ef00-f381-11ea-8bd2-1bd4c83666f3.png)

**Comment by MayuriNaik** [comment]
Closing this bug as it is discussed that it’s a regular occurrence for screen readers to read headers two times. It is likely to screen readers are reading the ARIA label and the content, which is why we get multiple readings.

**Comment by craxal** [comment]
This bug was closed mistakenly. The bug that should have been closed is #3509.

**Comment by craxal** [comment]
@MayuriNaik Please provide more information on how you navigated to the table and in the table. I am having trouble understanding and reproducing this issue.

**Comment by MayuriNaik** [comment]
Verified the bug in app version: 1.16.0, iOS: 10.15.7. The bug is fixed and hence closing the bug. PFA
[3508_Closed.mov.zip](https://github.com/microsoft/AzureStorageExplorer/files/5406469/3508_Closed.mov.zip)

**Comment by Ashutoshsahoo164** [comment]
GitHubTags : #Closed
